docs/collections/documents/add-document.md

```diff
@@ -0,0 +1,110 @@
+---
+title: Add Documents to Collection
+sidebar_position: 30
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+This page explains how to add a document to a Document Store collection.
+
+By default, a new document has only one attribute, `_key`. Defining the document `_key` attribute is optional. If you do not specify a key, then it will be autogenerated.
```

Comment:
Now that I reread this it might be a bit confusing as it could imply that only one attribute is optional, as opposed to having only one attribute period.  Feel free to disregard this one.